### PR TITLE
disable default trigger

### PR DESCRIPTION
**Pull request by p3dr0rv**
YAML pipelines are configured by default with a CI trigger on all branches.
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#ci-triggers

Disabling the CI trigger
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#disabling-the-ci-trigger

### CODE REVIEW

**Review by iambmelt — APPROVED** [review]

**Comment by iambmelt** [comment]
Recommend getting a second review from @shoatman

**Review by shoatman — APPROVED** [review]

**Pull request merged**
